integration/v3_auth_test.go

```diff
@@ -109,9 +109,23 @@ func TestV3AuthRevision(t *testing.T) {
 // TestV3AuthWithLeaseRevokeWithRoot ensures that granted leases
 // with root user be revoked after TTL.
 func TestV3AuthWithLeaseRevokeWithRoot(t *testing.T) {
+	testV3AuthWithLeaseRevokeWithRoot(t, false)
+}
+
+// TestV3AuthWithLeaseRevokeWithRootJWT creates a lease with a JWT-token enabled cluster.
+// And tests if server is able to revoke expiry lease item.
+func TestV3AuthWithLeaseRevokeWithRootJWT(t *testing.T) {
+	testV3AuthWithLeaseRevokeWithRoot(t, true)
+}
+
+func testV3AuthWithLeaseRevokeWithRoot(t *testing.T, jwt bool) {
```

Comment:
Rather than adding a flag here, perhaps it would be better to just pass in the `ClusterConfig` that should be used to run the test? If we find more edge cases in the future, it would be easier to add new configurations for the scenario.